<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportAnimationDialog</class>
 <widget class="QDialog" name="ExportAnimationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Animation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog, QWidget {
  background-color: #2E2E2E;  /* your “almost black” */
  color:            #FFFFFF;  /* ensure text is readable */
}

/* ── Default raised-button style for all QPushButtons ── */
QPushButton {
  background: qlineargradient(
    x1:0, y1:0, x2:0, y2:1,
    stop:0 #4A4A4A,
    stop:1 #2E2E2E
  );
  border:        1px solid #6A6A6A;
  border-radius: 4px;
  padding:       6px 12px;
  color:         #FFFFFF;
}

/* hover state */
QPushButton:hover {
  background: qlineargradient(
    x1:0, y1:0, x2:0, y2:1,
    stop:0 #5A5A5A,
    stop:1 #3A3A3A
  );
}

/* pressed state */
QPushButton:pressed {
  background: qlineargradient(
    x1:0, y1:0, x2:0, y2:1,
    stop:0 #333333,
    stop:1 #1E1E1E
  );
}

/* Disabled QPushButton */
QPushButton:disabled {
  background: #1C1C1C;
  color: #777777;
  border: 1px solid #444444;
}

/* QSpinBox text + border */
QSpinBox {
  background-color: #2E2E2E;
  color: #FFFFFF;
  border: 1px solid #6A6A6A;
  border-radius: 4px;
  padding: 4px;
}

/* Disabled QSpinBox */
QSpinBox:disabled {
  background-color: #1C1C1C;
  color: #777777;
  border: 1px solid #444444;
}

/* QComboBox (Dropdown) */
QComboBox {
  background-color: #2E2E2E;
  color: #FFFFFF;
  border: 1px solid #6A6A6A;
  border-radius: 4px;
  padding: 4px 30px 4px 8px; /* space for arrow */
}

/* Drop-down arrow */
QComboBox::drop-down {
  subcontrol-origin: padding;
  subcontrol-position: top right;
  width: 24px;
  border-left: 1px solid #6A6A6A;
  background-color: #3A3A3A;
}

/* Arrow icon */
QComboBox::down-arrow {
  image: url(:/icons/arrow_down_white.png); /* Optional: use Qt resource or SVG */
  width: 12px;
  height: 12px;
}

/* Disabled QComboBox */
QComboBox:disabled {
  background-color: #1C1C1C;
  color: #777777;
  border: 1px solid #444444;
}

/* Disabled drop-down arrow */
QComboBox::drop-down:disabled {
  background-color: #2A2A2A;
  border-left: 1px solid #444444;
}

/* Optional: grayed-out arrow icon for disabled combo boxes */
QComboBox::down-arrow:disabled {
  image: url(:/icons/arrow_down_gray.png); /* or omit for no arrow */
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="typeComboBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="formatLabel">
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="formatComboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportAnimationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportAnimationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
